frontend/public/style/_overrides.scss

```diff
@@ -1,3 +1,57 @@
+//reverted variations to regular property declarations for pf button component
+.pf-c-button {
+  &.pf-m-primary {
+    color: var(--pf-c-button--m-primary--Color);
+    background-color: var(--pf-c-button--m-primary--BackgroundColor);
+  }
+  &.pf-m-secondary {
+    color: var(--pf-c-button--m-secondary--Color);
+    background-color: var(--pf-c-button--m-secondary--BackgroundColor);
+  }
+  &.pf-m-tertiary {
+    color: var(--pf-c-button--m-tertiary--Color);
+    background-color: var(--pf-c-button--m-tertiary--BackgroundColor);
+  }
+  &.pf-m-link {
+    color: var(--pf-c-button--m-link--Color);
+    background-color: var(--pf-c-button--m-link--BackgroundColor);
+  }
+  &.pf-m-danger {
+    color: var(--pf-c-button--m-danger--Color);
+    background-color: var(--pf-c-button--m-danger--BackgroundColor);
+    &.pf-m-secondary {
+      --pf-c-button--m-danger--Color: var(--pf-c-button--m-secondary--m-danger--Color);
+      --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-secondary--m-danger--BackgroundColor);
+    }
+    &.pf-m-link {
+      --pf-c-button--m-danger--Color: var(--pf-c-button--m-link--m-danger--Color);
+      --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--BackgroundColor);
+    }
+  }
+  &.pf-m-warning {
+    color: var(--pf-c-button--m-warning--Color);
+    background-color: var(--pf-c-button--m-warning--BackgroundColor);
+  }
+  &.pf-m-control {
+    color: var(--pf-c-button--m-control--Color);
+    background-color: var(--pf-c-button--m-control--BackgroundColor);
+  }
+  &.pf-m-plain {
+    color: var(--pf-c-button--m-plain--Color);
+    background-color: var(--pf-c-button--m-plain--BackgroundColor);
+  }
+  // default disable styles so they override the overrides above
+  &:disabled,
+  &.pf-m-disabled,
+  &.pf-m-aria-disabled {
+    &::after {
```

Comment:
Looking at this again, you shouldn't need this `&::after {}` block.

Reply:
updated to remove that block